orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationChatResponse.java

```diff
@@ -107,4 +109,37 @@ public LLMChoice getChoice() {
         .getChoices()
         .get(0);
   }
+
+  /**
+   * Transform a JSON response into an entity of the given type.
+   *
+   * <p>This is possible on a request with a {@link OrchestrationTemplate#withJsonSchemaResponse}
+   * configured into {@link OrchestrationModuleConfig#withTemplateConfig}.
+   *
+   * @param type the class type to deserialize the JSON content into.
+   * @return the deserialized entity of type T.
+   * @param <T> the type of the entity to deserialize to.
+   */
+  @Nonnull
+  public <T> T entity(@Nonnull final Class<T> type) {
```

Comment:
(Minor)

* JavaDoc, method requires `@throws` declaration
* Please avoid external URLs in source-code, we're not going to check whether links remain healthy.
* Should exception message contain target type reference, or will it come up in the cause anyway?